R/zarr-array.R

```diff
@@ -80,7 +80,7 @@ ZarrArray <- R6::R6Class("ZarrArray",
     #' @keywords internal
     oindex = NULL,
     #' @description
-    #' (Re)load metadata from store.
+    #' (Re)load metadata from store without synchronization (file locking).
```

Comment:
I added this sort of documentation of synchronization (file locking) in a few places.